app/scripts/controllers/swaps.js

```diff
@@ -91,7 +91,8 @@ export default class SwapsController {
 
     this.indexOfNewestCallInFlight = 0
 
-    this.ethersProvider = new ethers.providers.Web3Provider(provider)
+    // The network ID is hard-coded as "1" because swaps is only used on Mainnet
```

Comment:
The chain ID 😅

Reply:
Is it though? `ethers` calls it a `aNetworkish`, so I had thought it was a network ID.

Reply:
Yeah, but when it's a number, it's the chain ID: https://docs.ethers.io/v5/api/providers/types/#providers-Networkish

Reply:
Thanks! I had actually looked for that in the docs and somehow wasn't able to find it.

Reply:
This has been done